docs/api.md

```diff
@@ -1,4 +1,4 @@
-# Puppeteer API v<!-- GEN:version -->0.9.0<!-- GEN:stop-->
+# Puppeteer API (<!-- GEN:version -->master<!-- GEN:stop-->)
```

Comment:
Remove the <!-- GEN:version --> and <!-- GEN:stop --> so that preprocessor doesn't mess with the file